app/mailers/comments_summary_mailer.rb

```diff
@@ -0,0 +1,16 @@
+class CommentsSummaryMailer < ApplicationMailer
+  # def summary_post_owner(post, owner, new_comments)
+  def summary_post_owner(owner)
+    # @new_comments = new_comments
+    @owner = User.find(owner)
+    @comments = Comment.all.where(["updated_at >= :date", {date: Date.today}])
+
+    @updates = []
+    @comments.each do |comment|
+      if comment.user_id == owner
+        @updates.push(comment)
+      end
+    end
```

Comment:
try this... if you are going to return an array, use the `.map` method

but, where is `@updates` and `@comments` being used